usefulUtilities/sitV2/appResources/appData/sitServerV2.js

```diff
@@ -157,13 +156,44 @@
     }
 
     // #endregion CAN SIT ZONE
+
+    /* Check children of the sit cube to make sure there is only one zone. If more are found, delete them */
+    function checkForExtraZones() {
+        var canSitZones = [];
+        Entities.getChildrenIDs(_this.entityID).forEach(function(childOfSitCube) {
+            var name = Entities.getEntityProperties(childOfSitCube, 'name').name;
+            if (name && name.indexOf("canSitZone") > -1) {
+                canSitZones.push(childOfSitCube);
+            }
+        });
+        if (canSitZones.length > 1) {
```

Comment:
I'm confused about what this is doing. Why can't you do the `Entities.deleteEntity()` call in the `forEach` above? Why do you need an extra `canSitZones` list? Why aren't you referencing `_this.canSitZoneID` to avoid deleting that one?

Reply:
That would be better...changing it.